src/pages/home/report/ReactionList/PopoverReactionList.js

```diff
@@ -2,15 +2,58 @@ import React from 'react';
 import {Dimensions} from 'react-native';
 
 import lodashGet from 'lodash/get';
+import lodashMap from 'lodash/map';
+import lodashFilter from 'lodash/filter';
+import lodashFind from 'lodash/find';
+import lodashEach from 'lodash/each';
+import lodashIsEqual from 'lodash/isEqual';
+
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import * as Report from '../../../../libs/actions/Report';
 import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
 import PopoverWithMeasuredContent from '../../../../components/PopoverWithMeasuredContent';
 
 import BaseReactionList from './BaseReactionList';
+import compose from '../../../../libs/compose';
+import reportActionPropTypes from '../reportActionPropTypes';
+import ONYXKEYS from '../../../../ONYXKEYS';
+import getPreferredEmojiCode from '../../../../components/Reactions/getPreferredEmojiCode';
+import withCurrentUserPersonalDetails from '../../../../components/withCurrentUserPersonalDetails';
+import * as PersonalDetailsUtils from '../../../../libs/PersonalDetailsUtils';
+import emojis from '../../../../../assets/emojis';
 
 const propTypes = {
+    /** Actions from the ChatReport */
+    reportActions: PropTypes.shape(reportActionPropTypes),
+
     ...withLocalizePropTypes,
 };
 
+const defaultProps = {
+    reportActions: {},
+};
+
+/**
+ * Given an emoji object and a list of senders it will return an
+ * array of emoji codes, that represents all used variations of the
+ * emoji.
+ * @param {{ name: string, code: string, types: string[] }} emoji
+ * @param {Array} users
+ * @return {string[]}
+ * */
+const getUniqueEmojiCodes = (emoji, users) => {
```

Comment:
This method is now defined twice and needs to be DRYed up https://github.com/Expensify/App/blob/main/src/components/Reactions/ReportActionItemReactions.js#L24